Externals.cfg

```diff
@@ -8,12 +8,8 @@ required = True
 
 [ufs_utils]
 protocol = git
-#repo_url = https://github.com/JeffBeck-NOAA/UFS_UTILS
-repo_url = https://github.com/NCAR/UFS_UTILS
-# Specify either a branch name or a hash but not both.
-#branch = feature/regional_release
-branch = feature/regional_release_STRING
-#hash = e5419633
+repo_url = https://github.com/JeffBeck-NOAA/UFS_UTILS
+branch = feature/regional_release
```

Comment:
So we're pointing to the HEAD of feature/regional_release, not a tag?

Reply:
I agree that pointing to the head of this branch is dangerous given the active nature of that PR; I will go ahead and replace that with a hash.

Reply:
Ok, thanks.

Reply:
I thought we were supposed to be pointing to Jeff's tag: v2.0alphav01

Reply:
I used this in my test on Monday
repo_url = https://github.com/JeffBeck-NOAA/UFS_UTILS
tag = v2.0alpha01

Reply:
I did not remember this Jamie, thanks for pointing it out. v2.0alpha01 is the same hash as the head of feature/regional_release so the results will be the same. I'll change the Externals.cfg to use that tag